openapi/paths/risk-score-simulation-jobs.yaml

```
@@ -0,0 +1,61 @@
+get:
+  x-badge: Experimental
+  x-products:
+    - Core
+  tags:
+    - Risk score
+  summary: Retrieve risk score simulation jobs
+  operationId: GetRiskScoreSimulationJobCollection
+  x-sdk-operation-name: getAll
+  security:
+    - SecretApiKey: []
+    - JWT: []
+  description: Retrieve risk score simulation jobs.
```

nitpick: [descriptions](https://github.com/Rebilly/rebilly/blob/main/website/api-definitions/WRITING-STYLE.md#descriptions) should start with an active verb

nitpick: based on our current way of writing 'getAll' type operations, I think we mention 'list' in this description 

[example](https://www.rebilly.com/docs/dev-docs/api/transactions/gettransactioncollection)

```suggestion
  description: Retrieves a list risk score simulation jobs.
```